reference/speaker.md

```diff
@@ -60,106 +60,59 @@ It is possible to stop all the sounds currently playing in a speaker by setting
 
-The `wb_speaker_speak` function allows the user to execute text-to-speech on the speaker. The value of the `text` parameter is converted into sound by Webots using the language specified by the `wb_speaker_set_language` function. The resulting sound is played through the speaker. The specified text could be plain text including punctuation signs such as "Hello world!", or can be enriched with special effects to make it more realistic. Such effects are speficied with XML tags.
+The `wb_speaker_get_engine` function allows the user to get the engine of the text-to-speech for a speaker device.
 
-**Text-to-speech XML tags**
+The `wb_speaker_set_language` function allows the user to set the language of the text-to-speech engine. For the `"pico"` engine, the `language` parameter should be set to one of the following values:
 
-- `ignore`: A text portion marked by `<ignore> ... </ignore>` is fully ignored by the synthesis. Ignored sections may be nested. Example:
+  - `"en-US"` for American English (default value).
+  - `"en-UK"` for British English.
+  - `"de-DE"` for German.
+  - `"es-ES"` for Spanish.
+  - `"fr-FR"` for French.
+  - `"it-IT"` for Italian.
 
-    ```xml
-Hello <ignore> any text </ignore> Mister Smith.
-    ```
+For the `"microsoft"` engine, it should follow the same format and correspond to an existing language, installed on the Windows computer.
+The format is `"ll-CC"` where `ll` (lowercase) corresponds to an ISO 639-1 language code and `CC` (uppercase) corresponds to an ISO 3166 country code, like for example `"en-US"` or `"fr-FR"`.
 
-    This example is equivalent to sending "Hello Mister Smith." to the text-to-speech engine.
+The `wb_speaker_get_language` function allows the user to get the language of the text-to-speech for a speaker device.
 
-- `p` of `paragraph`: Paragraph structures can be marked by `<p> ... </p>` or its extended form `<paragraph> ... </paragraph>`. In most cases, Webots automatically detects paragraph structures. The `<p>` tag can be used to enforce the setting of a paragraph structure. Example:
+The `wb_speaker_speak` function allows the user to execute text-to-speech on the speaker. The value of the `text` parameter is converted into sound by Webots using the engine specified by the `wb_speaker_set_engine` function and the language specified by the `wb_speaker_set_language` function. The resulting sound is played through the speficied `speaker`. The specified text could be plain text including punctuation signs such as "Hello world!", or can be enriched with special effects to make it more realistic. Such effects are specified with XML tags compliant with the SSML (Speech Synthesis Markup Language) standard. Here is a list of SSML tags that are supported by both the `pico` and the `microsoft` engines. Additional tags and parameters may be supported by the `microsoft` engine. Please refer to the Microsoft Windows SAPI5 documentation about it.
```

Comment:
We could maybe add a link to the Microsoft Windows SAPI5 documentation after mentioning it.